examples/reference/.env

```diff
@@ -1,5 +1,5 @@
 # Tide API
-CONTENT_API_SERVER=https://develop.content.reference.sdp.vic.gov.au/
+CONTENT_API_SERVER=https://nginx-php.pr-132.content-reference-sdp-vic-gov-au.sdp4.sdp.vic.gov.au
```

Comment:
Can you revert this change back to develop.content

Reply:
Done @waitingallday Ta